src/main/java/com/lyttledev/lyttleessentials/commands/FlyCommand.java

```diff
@@ -21,25 +21,22 @@ public FlyCommand(LyttleEssentials plugin) {
 
     @Override
     public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
+        if (!sender.hasPermission("lyttleessentials.fly")) {
+            Message.sendMessage(sender, "no_permission");
+            return true;
+        }
+
         if (!(sender instanceof Player) && args.length != 1) {
             Message.sendMessage(sender, "fly_usage");
             return true;
         }
 
         if (args.length > 1) {
-            if (!sender.hasPermission("lyttleessentials.fly")) {
-                Message.sendMessage(sender, "no_permission");
-                return true;
-            }
             Message.sendMessage(sender, "fly_usage");
             return true;
         }
 
         if (args.length == 0) {
-            if (!sender.hasPermission("lyttleessentials.fly.self")) {
```

Comment:
Please, keep the permissions: lyttleessentials.fly.self & lyttleessentials.fly.other.
Can you implement?

Reply:
Done